ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -587,7 +587,7 @@ private DependencyObject ResolveView(int tag)
             return view;
         }
 
-        private IViewManager ResolveViewManager(int tag)
+        public IViewManager ResolveViewManager(int tag)
```

Comment:
>public [](start = 8, length = 6)

Hmm, I wonder if we should go one step further on Android and create a simple interface that exposes only the methods that we want exposed (e.g., ResolveView and ResolveViewManager). Thoughts @matthargett?